python/cuml/tests/dask/test_dask_logistic_regression.py

```diff
@@ -491,3 +544,77 @@ def test_elasticnet(
     strength = 1.0 / lr.C
     assert l1_strength == lr.l1_ratio * strength
     assert l2_strength == (1.0 - lr.l1_ratio) * strength
+
+
+@pytest.mark.mg
+@pytest.mark.parametrize("fit_intercept", [False, True])
+@pytest.mark.parametrize(
+    "regularization",
+    [
+        ("none", 1.0, None),
+        ("l2", 2.0, None),
+        ("l1", 2.0, None),
+        ("elasticnet", 2.0, 0.2),
+    ],
+)
+@pytest.mark.parametrize("datatype", [np.float32])
+@pytest.mark.parametrize("delayed", [True])
+@pytest.mark.parametrize("n_classes", [2, 8])
+def test_sparse_from_dense(
+    fit_intercept, regularization, datatype, delayed, n_classes, client
+):
+    penalty = regularization[0]
+    C = regularization[1]
+    l1_ratio = regularization[2]
+
+    test_lbfgs(
+        nrows=1e5,
+        ncols=20,
+        n_parts=2,
+        fit_intercept=fit_intercept,
+        datatype=datatype,
+        delayed=delayed,
+        client=client,
+        penalty=penalty,
+        n_classes=n_classes,
+        C=C,
+        l1_ratio=l1_ratio,
+        convert_to_sparse=True,
+    )
+
+
+@pytest.mark.parametrize("dtype", [np.float32])
+def test_sparse_nlp20news(dtype, nlp_20news, client):
+    # sklearn score with max_iter = 10000
+    sklearn_score = 0.878
```

Comment:
This could become a maintenance nightmare should sklearn ever get updated. Any reason we can't just compute this in the test?

Reply:
Sounds Good! Already revised the test test to compute sklearn accuracy in the test.